src/database/migrations/1638217974847-UserTable.ts

```diff
@@ -0,0 +1,46 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class UserTable1638217974847 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'users',
+        columns: [
+          {
+            name: 'id',
+            type: 'int4',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'timestamp',
```

Comment:
remova and add `created_At` and `updated_at`
https://stackoverflow.com/questions/64941148/node-js-add-created-at-and-updated-at-in-entity-of-typeorm